hcx-apis/src/test/java/org/swasth/hcx/controllers/v1/JWTControllerTests.java

```diff
@@ -52,7 +52,7 @@ void participant_generate_token_success() throws Exception {
                 .addHeader("Content-Type", "application/json"));
         MvcResult mvcResult = mockMvc.perform(post(Constants.VERSION_PREFIX + Constants.PARTICIPANT_GENERATE_TOKEN)
                         .header("Content-Type", "application/x-www-form-urlencoded")
-                        .param("username","hcxgateway@gmail.com")
+                        .param("username","testprovider1@apollo.com")
```

Comment:
Take it from the test configuration file. It will be easy to make changes in the future.

Reply:
Usernames are different for each test case. Username is from mock data, making configuration is not required.